packages/core/src/browser/label-provider.ts

```diff
@@ -82,12 +82,51 @@ export interface LabelProviderContribution {
      */
     affects?(element: object, event: DidChangeLabelEvent): boolean;
 
+    getUriLabel?(element: object): string;
```

Comment:
`getLongName` should be used to get human readable name, there should not be new APIs

Reply:
done